test/Passes/DefaultPipeline/default-tpp-passes.mlir

```diff
@@ -206,3 +206,15 @@ func.func @mlp(%arg0: tensor<128x256xf32>, %arg1: tensor<256x512xf32>,
 
   return %3 : tensor<128x512xf32>
 }
+
+// -----
+
+// CHECK-LABEL: batch_matmul_rewrite
+func.func @batch_matmul_rewrite(%arg0: tensor<512x32x64xf32>, %arg1: tensor<512x64x32xf32>) -> tensor<512x32x32xf32> {
+  %0 = tensor.empty() : tensor<512x32x32xf32>
+  // CHECK: scf.parallel
+  // CHECK: xsmm_brgemm_invoke
```

Comment:
Shouldn't this be `xsmm.gemm` given this pass?

Reply:
The GEMM get's packed and mapped to a BRGEMM. The sequence here is: linalg.batch_matmul -> linalg.matmul -> generic (blocked matmul) -> tpp.brgemm